custom_components/moonraker/__init__.py

```diff
@@ -109,10 +109,16 @@ async def _async_update_data(self):
             METHOD.PRINTER_OBJECTS_QUERY, self.query_obj
         )
         info = await self.async_fetch_data(METHOD.PRINTER_INFO)
+        history = await self.async_fetch_data(METHOD.SERVER_HISTORY_TOTALS)
```

Comment:
This will always be called and thus always print an error message in the log-in debug if history is not available. Not an exception but an method not found error